### PR TITLE
Team Welcome and Team Tour changes

### PR DESCRIPTION
**Pull request by pkcodes92**
All the changes made in the past two weeks

### CODE REVIEW

File: Source/YICG.Apps.Teams.DigitalKnowBot/YICG.Apps.Teams.DigitalKnowBot/Bots/GangaGameBot.cs [hunk 1]
```diff
@@ -21,14 +24,17 @@ namespace YICG.Apps.Teams.DigitalKnowBot.Bots
     public class GangaGameBot : ActivityHandler
     {
         private readonly string appBaseUri;
+        private readonly MicrosoftAppCredentials microsoftAppCredentials;
```
Comment (pkcodes92) [inline Source/YICG.Apps.Teams.DigitalKnowBot/YICG.Apps.Teams.DigitalKnowBot/Bots/GangaGameBot.cs:27]: I'm adding the Microsoft App Credentials as a readonly

File: Source/YICG.Apps.Teams.DigitalKnowBot/YICG.Apps.Teams.DigitalKnowBot/Cards/TourCarousel.cs [hunk 2]
```diff
@@ -34,6 +34,26 @@ public static IEnumerable<Attachment> GetUserTourCards(string appBaseUri)
             };
         }
 
+        /// <summary>
+        /// This method will generate the team tour carousel.
+        /// </summary>
+        /// <param name="appBaseUri">The local address of the bot.</param>
+        /// <returns>A type of <see cref="IEnumerable{Attachment}"/> which will allow for the carousel tour to be shown.</returns>
+        public static IEnumerable<Attachment> GetTeamTourCards(string appBaseUri)
```
Comment (apatnam08) [inline Source/YICG.Apps.Teams.DigitalKnowBot/YICG.Apps.Teams.DigitalKnowBot/Cards/TourCarousel.cs:42]: Creating a _team_ tour card carousel

File: Source/YICG.Apps.Teams.DigitalKnowBot/YICG.Apps.Teams.DigitalKnowBot/appsettings.json [hunk 3]
```diff
@@ -5,5 +5,6 @@
   "StorageConnectionString": "",
   "ScoreThreshold": "0.5",
   "AccessCacheExpiryInDays": 5,
-  "AppBaseUri": ""
+  "AppBaseUri": "",
+  "ExpertsTeamId": ""
```
Comment (apatnam08) [inline Source/YICG.Apps.Teams.DigitalKnowBot/YICG.Apps.Teams.DigitalKnowBot/appsettings.json:9]: Added a new part to the appsettings.json file, the ExpertsTeamId.

**Review by Chocolate-Jeff — APPROVED** [review]
I approve

File: Source/YICG.Apps.Teams.DigitalKnowBot/YICG.Apps.Teams.DigitalKnowBot/Bots/GangaGameBot.cs [hunk 4]
```diff
@@ -205,7 +205,7 @@ private async Task OnMessageActivityInChannelAsync(IMessageActivity message, ITu
             {
                 case Constants.TeamTourChannelCommand:
                     var teamTourCards = TourCarousel.GetTeamTourCards(this.appBaseUri);
-                    await turnContext.SendActivityAsync(MessageFactory.Carousel(teamTourCards));
+                    await turnContext.SendActivityAsync(MessageFactory.Carousel(teamTourCards), cancellationToken);
```
Comment (pkcodes92) [inline Source/YICG.Apps.Teams.DigitalKnowBot/YICG.Apps.Teams.DigitalKnowBot/Bots/GangaGameBot.cs:208]: Always make sure to add the cancellationToken

File: Source/YICG.Apps.Teams.DigitalKnowBot/YICG.Apps.Teams.DigitalKnowBot/Bots/GangaGameBot.cs [hunk 5]
```diff
@@ -198,7 +204,8 @@ private async Task OnMessageActivityInChannelAsync(IMessageActivity message, ITu
             switch (text)
             {
                 case Constants.TeamTourChannelCommand:
-                    await turnContext.SendActivityAsync(MessageFactory.Text("Cool, you want to know how I help team! I'll show you in a minute!"), cancellationToken);
+                    var teamTourCards = TourCarousel.GetTeamTourCards(this.appBaseUri);
+                    await turnContext.SendActivityAsync(MessageFactory.Carousel(teamTourCards));
                     break;
                 default:
                     await turnContext.SendActivityAsync(MessageFactory.Text("Ooook... My 🤖 🧠 cannot understand!!!"), cancellationToken);
```
Comment (apatnam08) [inline Source/YICG.Apps.Teams.DigitalKnowBot/YICG.Apps.Teams.DigitalKnowBot/Bots/GangaGameBot.cs:211]: Later on, try to make the writing more formal (**NO EMOJIS!**)

**Review by apatnam08 — APPROVED** [review]
I APPROVE!

File: Source/YICG.Apps.Teams.DigitalKnowBot/YICG.Apps.Teams.DigitalKnowBot/Bots/GangaGameBot.cs (hunk 5 above)
Comment (pkcodes92) [reply Source/YICG.Apps.Teams.DigitalKnowBot/YICG.Apps.Teams.DigitalKnowBot/Bots/GangaGameBot.cs:211]: Aw dang! I'll have the emojis back though!

**Pull request merged**